src/test/java/com/google/flourbot/api/DriveCloudSheetTest.java

```diff
@@ -0,0 +1,94 @@
+package com.google.flourbot.api;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+import java.io.IOException;
+import java.security.GeneralSecurityException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+
+@RunWith(JUnit4.class)
+public final class DriveCloudSheetTest {
+
+  // Create a sheet where in every cell A1:E5, the cell contents is the location
+  // Ex: The top left cell at A1 will contain the string "A1"
+  // Remember to give the bot permission to edit the sheet
+  private static final String SPREADSHEET_ID = "";
+  private static final String SHEET_NAME = "Sheet1";
+
+  private static final String RANGE_SINGLE = "B1:B1";
+  private static final String RANGE_SINGLE_EMPTY = "F1:F1";
+  private static final String RANGE_BLOCK = "A1:C3";
+  private static final String RANGE_BLOCK_EMPTY = "G2:H4";
+  private static final String RANGE_ROW = "2";
+  private static final String RANGE_COLUMN = "C";
+  //TODO: add invalid range options
+  
+  private CloudDocClient cloudDocClient;
+  private CloudSheet cloudSheet;
+
+  @Before
+  public void setUp() {
+    cloudDocClient = new DriveClient();
+    try {
+      cloudSheet = cloudDocClient.getCloudSheet(SPREADSHEET_ID);
+    } catch (IOException e) {
```

Comment:
I don't understand what is the reason for this catches. In general, your test should be stable enough, so this never happens. Also, setup should not have any assertions - the fail will be communicated with the exception anyway.

Reply:
Updated to handle exceptions in try catch in cloud doc client - so I think that should throw the proper exceptions and cause the fail without the try catch nor assertions in the setup?